### ISSUE TITLE
How to properly "shut down" velocity on removed element from the DOM

### ISSUE THREAD

**Issue by eylord** [julianshapiro/velocity#585]
Hello,

i've tried to find a clean solution to this but failed.

I've got temporary DOM elements that I animate with Velocity. At some point, I'm removing them from the DOM tree and it gives me velocity error the next frame. I've tried everything : 
- remove()
- removeData()
- detach()

Every time it seems that velocity is trying to do something on my element even tought it is removed and I removed the jquery Data linked to it.

I cannot just call "finish" to the animation since the element would be gone before the "stop" instruction can be executed.

TLDR : How can I "exit" an animation call instantly and cleanly before removing my DOM element ?

**Comment by Rycochet** [comment]
Detach then stop then forget?

**Comment by ydaniv** [comment]
Could you put this in a fiddle?

**Comment by kenwheeler** [comment]
Closing for no fiddle. If one is provided, I'll reopen.
